### ISSUE TITLE
New Menu too Uniform with Nodes

### ISSUE THREAD

**Issue by WASasquatch** [comfyanonymous/ComfyUI#379]
New menu theme is just blending with the nodes, so when you can an entire screen full of nodes, I tend to actually have to ***look*** for my menu. :P

**Comment by WASasquatch** [comment]
Not only do I lose the menu as looking like a node itself, but when zoomed in with lots of nodes, because it's color is identical to a node, can literally blend right into the nodes because it has no border, or box shadow to emphasize it as a floating element above the node plane.

**Comment by wyrde** [comment]
![menu 2023-04-05 18_44_10-Mozilla Firefox](https://user-images.githubusercontent.com/9657443/230227948-e3a32e07-13f1-42a1-8556-0fbafe30e895.png)

the menu's drop shadow is right/bottom, so moving it to the upper left corner makes it more obvious.
I do think the shadows can use a few more pixels width though. Also help for the "active node" glow.

**Comment by WASasquatch** [comment]
> ![menu 2023-04-05 18_44_10-Mozilla Firefox](https://user-images.githubusercontent.com/9657443/230227948-e3a32e07-13f1-42a1-8556-0fbafe30e895.png)
> 
> the menu's drop shadow is right/bottom, so moving it to the upper left corner makes it more obvious. I do think the shadows can use a few more pixels width though. Also help for the "active node" glow.

Probably should also not be a directional shadow since it is a positional menu you can put anywhere the user wants. But the menu in general should have a differentiating theme from the graph too. It's you main triggering point. Shouldn't have to look for it. Should be obvious.

**Comment by Davemane42** [comment]
replacing the the "Queue Prompt" button with a big orange "Generate" would differentiate the menu

**Comment by WASasquatch** [comment]
> replacing the the "Queue Prompt" button with a big orange "Generate" would differentiate the menu

Yeah, or just change the color theme of the button in general. I don't know about "generate" cause it is literally a queue button.

**Comment by wyrde** [comment]
Being able to toggle it to contrast with the current browser light/dark setting, maybe? eg, dark theme when the browser is light.

**Comment by WASasquatch** [comment]
> Being able to toggle it to contrast with the current browser light/dark setting, maybe? eg, dark theme when the browser is light.

Ooh, yeah that would be nice. A prefers-color-scheme or whatever is called env bar to determine colors

**Comment by robinjhuang** [comment]
@WASasquatch Did you try the new Beta UI? Is this still an issue?
https://github.com/comfyanonymous/ComfyUI/pull/3112

**Comment by WASasquatch** [comment]
I haven't tried the new beta UI. Hadn't heard about it. O.o

But it is still an issue with original UI. The menu has no bordering and
shares the same color as nodes.
